start

```diff
@@ -197,20 +196,4 @@ function soroban_rpc_status () {
   print_screen_output "soroban rpc reported ready status, the service can be used by tools/cli now ..." 
```

Comment:
Related: https://github.com/stellar/soroban-tools/pull/449

Reply:
we probably want to wait for this before merging

Reply:
we may want to wait for the PR above before merging

Reply:
yes, for `soroban_rpc_status` wait loop to work effectively in here, the rpc's `getHealth` http endpoint should only return healthy response if the service is past any panic paths, ingestion is in stable state(got a checkpoint). 

otherwise, there's an edge case where `getHealth` is up briefly while rpc is starting ingestion and before it panic's due to no checkpoint during which it could respond with healthy response from `getHealth`? Not sure if rpc's http service is up before or after the ingestion startup process in rpc?  

if rpc 's http service is only available after ingestion is success startup, then it should be ok to remove this now, since `soroban_rpc_status` would just continue to wait in retry loop on`getHealth`